test/index.ts

```diff
@@ -871,11 +871,11 @@ test('blockchain test', t => {
   })
 
   t.test('mismatched chains', st => {
-    const common = new Common('rinkeby')
+    const common = new Common('mainnet')
```

Comment:
Changing to `mainnet` as Muir Glacier is only currently active for PoW networks. OK